nifi-nar-bundles/nifi-standard-services/nifi-lookup-services-bundle/nifi-lookup-services/src/main/java/org/apache/nifi/lookup/RestLookupService.java

```diff
@@ -251,6 +264,8 @@ public void onEnabled(final ConfigurationContext context) {
         buildHeaders(context);
 
         urlTemplate = context.getProperty(URL);
+
+        responseHandler = context.getProperty(PROP_RESPONSE_CODE_HANDLING);
```

Comment:
After changing the member variable type, the `asAllowableValue()` method can be used to resolve the `enum` value from the property value.